qipr/registry/templates/registry/home.html

```diff
@@ -0,0 +1,144 @@
+<html>
+    <head>
+        {% load staticfiles %}
+        <meta name="viewport" content="width=device-width">
+        <link href="{% static 'qipr/css/materialize.css' %}" type="text/css" rel="stylesheet"/>
+        <link href="{% static 'qipr/css/main.css' %}" type="text/css" rel="stylesheet"/>
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Arvo|Open+Sans|Roboto+Condensed|Source+Sans+Pro" rel="stylesheet">
+        <link rel="icon" type="image/png" sizes="32x32" href="{% static 'qipr/img/favicon-32x32.png' %}">
+        <script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <nav>
+            <div class="nav-wrapper bg-blue">
+              <a href="home.html" class="brand-logo">
+                <img class="nav__logo" src="/static/qipr/img/qipr_outlines.svg">
```

Comment:
Please use the same {% %} django notation you used for the href links and icon above